server/Episememe.Application/DataTransfer/SearchMediaDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Episememe.Application.DataTransfer
+{
+    public class SearchMediaDto
+    {
+        public IEnumerable<string> IncludedTags { get; set; } = null!;
+        public IEnumerable<string> ExcludedTags { get; set; } = null!;
```

Comment:
Remove the null! It suppresses any compiler warnings regarding the fact that `IncludedTags` and `ExcludedTags` may be null and need to be handled